src/auth/isApplicationLoggedIn.ts

```diff
@@ -0,0 +1,36 @@
+import { FastifyAuthFunction } from "@fastify/auth";
+import { NotLoggedError } from "../errors/NotLoggedError";
+import * as jwt from "../jwt";
+import ApplicationModel from "../model/ApplicationModel";
+
+export const isApplicationLoggedIn = (async (req, res) => {
+    const token = req.headers["x-api-key"] as string;
+
+    console.log(req.headers);
+
+    if (!token) {
+        throw new NotLoggedError();
+    }
+
+    const jwtUser = await jwt.verify(
+        token,
+        process.env.APPLICATION_TOKEN_SECRET!,
+    );
+
+    if (!jwtUser.id) throw new NotLoggedError();
```

Comment:
```suggestion
    if (!jwtUser.id) {
        throw new NotLoggedError();
    }
```
Ou alors changer les autres ifs du meme genre pour que ce soit homogène